src/components/Tables/AppletsTable/Table/AppletItem/AppletItem.styles.ts

```diff
@@ -1,9 +1,38 @@
-import { Box, styled } from '@mui/material';
+import { Box, Button, styled } from '@mui/material';
+
+import theme from 'styles/theme';
 import { FolderApplet } from 'redux/modules';
+import { variables } from 'styles/variables';
 
 export const StyledAppletName = styled(Box)`
   display: flex;
   align-items: center;
   margin-left: ${({ applet }: { applet: FolderApplet }) =>
     applet?.depth ? `${applet.depth * 4.4}rem` : 0};
 `;
+
+export const StyledPin = styled(Button)`
+  width: 4rem;
+  height: 4rem;
+  min-width: 4rem;
+  padding: 0;
```

Comment:
Do we have Button without padding component?

Reply:
Replaced to StyledClearedButton